### PR TITLE
fix(configs): Cleanup SITARA's toc tree

### PR DESCRIPTION
**Pull request by cshilwant**
- There were files mentioned in device toc configs that no longer exists or are renamed.

- Used bin/check-configs.sh [1] to find out such files in configs for SITARA platforms & remove / rename them accordingly.

[1]: https://github.com/TexasInstruments/processor-sdk-doc/blob/master/bin/check-configs.sh

### CODE REVIEW

**Comment by cshilwant** [comment]
Preview available at https://cshilwant.github.io/processor-sdk-doc/

File: configs/AM437X/AM437X_linux_toc.txt [hunk 1]
```diff
@@ -72,6 +66,10 @@ linux/Foundational_Components_Kernel_Release_Notes
 linux/Foundational_Components_Kernel_RT_Release_Notes
 linux/Foundational_Components_Kernel_LTP-DDT_Validation
 linux/Foundational_Components_Kernel_FAQs
+linux/Foundational_Components/Power_Management/pm_overview
+linux/Foundational_Components/Power_Management/pm_dvfs
+linux/Foundational_Components/Power_Management/pm_cpuidle
+linux/Foundational_Components/Power_Management/pm_suspend_resume
```
Comment (jeevantelukula) [inline configs/AM437X/AM437X_linux_toc.txt:72]: @cshilwant In the preview, can't find the Power Management section in Foundational Components. But able to open the page using search functionality, and it is not highlighted in Table of Contents on the left side.
See - https://cshilwant.github.io/processor-sdk-doc/processor-sdk-linux-AM335X/esd/docs/09_03_05_02/linux/Foundational_Components/Power_Management/pm_suspend_resume.html

Comment (jeevantelukula) [reply configs/AM437X/AM437X_linux_toc.txt:72]: Same for AM335x_linux_toc

Comment (cshilwant) [reply configs/AM437X/AM437X_linux_toc.txt:73]: Good catch @jeevantelukula 

Updated the PR & I now see it in Table of contents - https://cshilwant.github.io/processor-sdk-doc/processor-sdk-linux-AM335X/esd/docs/09_03_05_02/linux/Foundational_Components.html#

**Comment by cshilwant** [comment]
@praneethbajjuri @sukrutb to review & see if Power Management sections which are added now are relevant for AM335x & AM437x

**Comment by cshilwant** [comment]
@jsuhaas22 to review changes done in debian toc tree

**Comment by cshilwant** [comment]
@makohoek to review changes done in android toc tree

**Review by jeevantelukula — DISMISSED** [review]

**Review by makohoek — APPROVED** [review]
For `configs/AM62X/AM62X_android_toc.txt`, LGTM :+1:

**Review by jsuhaas22 — DISMISSED** [review]
Debian part LGTM.

File: configs/AM62X/AM62X_linux_toc.txt [hunk 2]
```diff
@@ -13,7 +13,7 @@ devices/AM62X/linux/Release_Specific_Kernel_Performance_Guide
 devices/AM62X/linux/Linux_Performance_Guide
 devices/AM62X/linux/RT_Linux_Performance_Guide
 devices/AM62X/linux/Release_Specific_Supported_Platforms_and_Versions
-devices/AM62X/linux/Release_Specific_QSG
+devices/AM62X/linux/_Release_Specific_QSG
```
Comment (StaticRocket) [inline configs/AM62X/AM62X_linux_toc.txt:16]: Files starting with an `_` are only to be included. They should not be listed in TOC trees directly.

**Review by StaticRocket — CHANGES_REQUESTED** [review]

File: configs/AM62X/AM62X_linux_toc.txt (hunk 2 above)
Comment (cshilwant) [reply configs/AM62X/AM62X_linux_toc.txt]: @StaticRocket Updated the PR

**Review by jsuhaas22 — APPROVED** [review]

**Review by jeevantelukula — APPROVED** [review]

**Comment by cshilwant** [comment]
@StaticRocket Addressed your comments

File: configs/AM62AX/AM62AX_linux_toc.txt [hunk 3]
```diff
@@ -14,7 +14,6 @@ devices/AM62AX/linux/Release_Specific_Kernel_Performance_Guide
 devices/AM62AX/linux/Linux_Performance_Guide
 devices/AM62AX/linux/RT_Linux_Performance_Guide
 devices/AM62AX/linux/Release_Specific_Supported_Platforms_and_Versions
-devices/AM62AX/linux/Release_Specific_QSG
```
Comment (sinha-shreyash) [inline configs/AM62AX/AM62AX_linux_toc.txt:17]: @cshilwant , any specific reason for removing this, I see it is present under this directory with name _Release_Specific_QSG.rst

File: configs/AM62AX/AM62AX_linux_toc.txt [hunk 4]
```diff
@@ -23,12 +22,10 @@ linux/Overview_Technical_Support
 linux/Overview_GPLv3_Disclaimer
 linux/Overview/GCC_ToolChain
 linux/Overview/Processor_SDK_Linux_create_SD_card
-linux/Overview/Processor_SDK_Building_The_SDK
```
Comment (sinha-shreyash) [inline configs/AM62AX/AM62AX_linux_toc.txt:26]: same for this

File: configs/AM62AX/AM62AX_linux_toc.txt (hunk 3 above)
Comment (StaticRocket) [reply configs/AM62AX/AM62AX_linux_toc.txt:17]: https://github.com/TexasInstruments/processor-sdk-doc/pull/132#discussion_r1889001624

File: configs/AM62AX/AM62AX_linux_toc.txt (hunk 4 above)
Comment (StaticRocket) [reply configs/AM62AX/AM62AX_linux_toc.txt:26]: https://github.com/TexasInstruments/processor-sdk-doc/pull/132#discussion_r1889001624

**Review by StaticRocket — APPROVED** [review]

**Pull request merged**